src/Orc.Controls/Orc.Controls.Shared/Controls/DateTimePicker/Views/DateTimePickerControl.xaml.cs

```diff
@@ -46,17 +53,39 @@ public DateTimePickerControl()
                 NumericTBSecond,
             };
 
-            NumericTBDay.RightBoundReached += NumericTextBoxOnRightBoundReached;
-            NumericTBMonth.RightBoundReached += NumericTextBoxOnRightBoundReached;
-            NumericTBYear.RightBoundReached += NumericTextBoxOnRightBoundReached;
-            NumericTBHour.RightBoundReached += NumericTextBoxOnRightBoundReached;
-            NumericTBMinute.RightBoundReached += NumericTextBoxOnRightBoundReached;
+            SubscribeNumericTextBoxes();
 
-            NumericTBYear.LeftBoundReached += NumericTextBoxOnLeftBoundReached;
-            NumericTBMonth.LeftBoundReached += NumericTextBoxOnLeftBoundReached;
-            NumericTBHour.LeftBoundReached += NumericTextBoxOnLeftBoundReached;
-            NumericTBMinute.LeftBoundReached += NumericTextBoxOnLeftBoundReached;
-            NumericTBSecond.LeftBoundReached += NumericTextBoxOnLeftBoundReached;
+            ApplyFormat();
```

Comment:
Just wanted to be compliant with existing code in DatePickerControl, but yes I'll fix it.